libkbfs/init.go

```diff
@@ -120,6 +125,7 @@ func AddFlags(flags *flag.FlagSet, ctx Context) *InitParams {
 	flag.Var(SizeFlag{&params.LogFileConfig.MaxSize}, "log-file-max-size", "Maximum size of a log file before rotation")
 	// The default is to *DELETE* old log files for kbfs.
 	flag.IntVar(&params.LogFileConfig.MaxKeepFiles, "log-file-max-keep-files", defaultParams.LogFileConfig.MaxKeepFiles, "Maximum number of log files for this service, older ones are deleted. 0 for infinite.")
+	flags.StringVar(&params.WriteJournalRoot, "write-journal-root-this-may-lose-data", "", "If non-empty, permits write journals to be turned on for TLFs which will be put in the given directory")
```

Comment:
Be explicit in the comment that this is experimental, in case someone glosses over the flag name.